src/super_gradients/training/transforms/transforms.py

```diff
@@ -1510,7 +1510,7 @@ def random_affine(
 
     M = get_affine_matrix(img.shape[:2], target_size, degrees, translate, scales, shear)
 
-    img = cv2.warpAffine(img, M, dsize=target_size, borderValue=(border_value, border_value, border_value))
+    img = cv2.warpAffine(img, M, dsize=target_size[::-1], borderValue=(border_value, border_value, border_value))
```

Comment:
I suggest to add an explicit message to a method's docstring to indicate whether `target_size` represents size as (rows, cols) or (cols, rows).
And unpack these to add more clarity to the code. Eg:
```
rows, cols = target_size[:2]
img = cv2.warpAffine(img, M, dsize=(cols, rows), ...)

```

Reply:
Good point, I've adjusted it.